samples/app/package.json

```diff
@@ -14,7 +14,7 @@
     "gradle": "./android/gradlew --project-dir=./android --no-daemon",
     "gradle:write-locks": "cd android && ./gradlew app:dependencies --write-locks",
     "prepare": "patch-package",
-    "ios:setup": "bundle install --frozen && cd ios && bundle exec pod install",
```

Comment:
We already have frozen set to true in our config file, so this deprecated flag is no longer needed